test/tests/bitcraftmini-pretest.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
```

Comment:
Is this file gonna be replaced with something, or do we not need it anymore?

Reply:
Yeah, so afaict it's not really needed anymore; it's skipped in CI and I think bitcraft mini isn't even really used for smoketesting stdb anymore.